test-case/check-suspend-resume.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+
+##
+## Case Name: check suspend/resume status
+## Preconditions:
+##    N/A
+## Description:
+##    Run the suspend/resume command to check device status
+## Case step:
+##    1. switch suspend/resume operation
+##    2. use rtcwake -m mem command to do suspend/resume
+##    3. check command return value
+##    4. check dmesg errors
+##    5. check wakeup increase
+## Expect result:
+##    suspend/resume recover
+##    check kernel log and find no errors
+##
+
+source $(dirname ${BASH_SOURCE[0]})/../case-lib/lib.sh
+
+random_min=3    # wait time should >= 3 for other device wakeup from sleep
+random_max=20
+def_power=$(sed 's/ /\n/g' /sys/power/mem_sleep|grep '\['|sed 's/\[//g;s/\]//g')
+
+OPT_OPT_lst['l']='loop'     OPT_DESC_lst['l']='loop count'
+OPT_PARM_lst['l']=1         OPT_VALUE_lst['l']=5
+
+OPT_OPT_lst['p']='power'    OPT_DESC_lst['p']="suspend/resume type from /sys/power/mem_sleep"
+OPT_PARM_lst['p']=1         OPT_VALUE_lst['p']="$def_power"
```

Comment:
can we make this default as empty string? And then only overwrite the setting when there are some values here.